crowbar_framework/app/models/api/crowbar.rb

```diff
@@ -112,27 +112,30 @@ def repocheck
         end
       end
 
-      def ceph_healthy?
+      def ceph_status
+        ret = {}
         ceph_node = NodeObject.find("roles:ceph-mon AND ceph_config_environment:*").first
-        return true if ceph_node.nil?
+        return ret if ceph_node.nil?
         ssh_retval = ceph_node.run_ssh_cmd("LANG=C ceph health 2>&1")
         unless ssh_retval[:stdout].include? "HEALTH_OK"
-          Rails.logger.warn("ceph cluster health check failed with #{ssh_retval[:stdout]}")
-          return false
+          ret["errors"] = [
+            "ceph cluster health check failed with #{ssh_retval[:stdout]}"
+          ]
         end
-        true
+        ret
```

Comment:
MINOR: again, for this cases where you're forced to explicitly return something, using `tap` is always a good option, as it always returns with it's block variable, whcih could be `ret`